config/settings/sandbox.yml

```diff
@@ -14,6 +14,20 @@ dfe_sign_in:
 
 features:
   basic_auth: false
+  enable_feedback_link: false
+  sync_from_dttp: false
```

Comment:
I think this flag is used for the schools import job, don't we want this on for the school routes/schools picker?

Reply:
Oh yeah. Good shout. I turned it off as the configured DTTP is `dttp-test` and I wasn't sure what we'd get from there. Not sure we've got the right creds in the secrets either. I don't want to sync from prod as real users will get synced too. I'll give it a go and see.

Reply:
I've updated @darokel . I checked the secrets and sandbox is configured for test DTTP. Let's give it a go and see what happens.